src/js/components/FilterInputText.js

```diff
@@ -92,15 +92,15 @@ class FilterInputText extends React.Component {
     });
 
     return (
-      <span className="form-control-group-add-on form-control-group-add-on-append">
+      <span className="form-control-group-add-on">
```

Comment:
I still see the `form-control-group-add-on-append` and `form-control-group-add-on-prepend` in the Form styles for reactjs-components.
Should these styles live here or there? If here, then we should probably remove them from reactjs-components or the other way around.